aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -20,10 +20,24 @@ proposals:
       title: "Enable Block Gas Limit"
       description: "AIP-33: The block gas limit is a feature that terminates block execution when the gas consumed by the committed prefix of transactions exceeds the limit, thus providing predictable latency."
       discussion_url: "https://github.com/aptos-foundation/AIPs/issues/132"
-    execution_mode: MultiStep
+    execution_mode: SingleStep
```

Comment:
Keep it as multistep please

Reply:
Is this causing the test failure or that's unrelated?

Reply:
Also, Can you explain what's the difference between two?

Reply:
SingleStep is executing one change, while MultiStep chain several updates together when proposing

Reply:
@movekevin why this one need to be multistep? it's just changing one config?

Reply:
Single step triggers a different code path in the cli and smart contract that should now be deprecated. Cc @runtian-zhou

Reply:
Interesting, in this case, shall we rename the the flow when deprecating? or just make it the default?

Reply:
Yes that's the plan. We have that listed for a while now - kill single step completely in the CLI flow. cc @thepomeranian